### PR TITLE
Doc link updated so that TAB will work

### PR DESCRIPTION
**Pull request by ANSHUMAN87**
When clink on link with Right Button from mouse and click on "Open Link in New Tab", it should open in the new TAB, but it throws error "Unknown type".

### CODE REVIEW

File: docs/tutorials/federated_learning_for_image_classification.ipynb [hunk 1]
```diff
@@ -767,7 +767,7 @@
         "\n",
         "In a nutshell, *federated computations* are programs in TFF's internal language\n",
         "that can express various federated algorithms (you can find more about this in\n",
-        "the [custom algorithms](custom_federated_algorithms_1.ipynb) tutorial). In this\n",
+        "the [custom algorithms](https://github.com/ANSHUMAN87/federated/blob/master/docs/tutorials/custom_federated_algorithms_1.ipynb) tutorial). In this\n",
```
Comment (michaelreneer) [inline docs/tutorials/federated_learning_for_image_classification.ipynb]: Typo maybe? Need to replace "ANSHUMAN87" with "tensorflow"

File: docs/tutorials/federated_learning_for_image_classification.ipynb [hunk 2]
```diff
@@ -82,7 +82,7 @@
         "This tutorial, and the Federated Learning API, are intended primarly for users\n",
         "who want to plug their own TensorFlow models into TFF, treating the latter\n",
         "mostly as a black box. For a more in-depth understanding of TFF and how to\n",
-        "implement your own federated learning algorithms, see the tutorials on the FC Core API - [Custom Federated Algorithms Part 1](custom_federated_algorithms_1.ipynb) and [Part 2](custom_federated_algorithms_2.ipynb).\n",
+        "implement your own federated learning algorithms, see the tutorials on the FC Core API - [Custom Federated Algorithms Part 1](https://github.com/tensorflow/federated/blob/master/docs/tutorials/custom_federated_algorithms_1.ipynb) and [Part 2](https://github.com/tensorflow/federated/blob/master/docs/tutorials/custom_federated_algorithms_2.ipynb).\n",
```
Comment (michaelreneer) [inline docs/tutorials/federated_learning_for_image_classification.ipynb]: Can you update this change to use the following links instead?

`https://github.com/tensorflow/federated/blob/v0.2.0/docs/tutorials/custom_federated_algorithms_1.ipynb`

Basically, we want the colabs to point at the latest release version, not master.

Comment (ANSHUMAN87) [reply docs/tutorials/federated_learning_for_image_classification.ipynb]: @michaelreneer : Thanks for your feedback, will certainly do the changes and upload, Thanks!

**Comment by ANSHUMAN87** [comment]
@michaelreneer : I have handled all notebooks similar issues, please check it, TIA!

**Comment by ANSHUMAN87** [comment]
> Sorry, one last request, based on timing. We are in the middle of releasing the next version of TFF v0.3.0, can you update these links to reflect that new version? Sorry, I thought we would be able to slide this change into the last release. I'll approve after that release. I don't think there would be any merge conflicts so I don't expect any other work there.

No problem, i can do that, would you be able to share one example link like you shared before, so that i can replace with proper one, otherwise there might be chance of broken link. Thanks!

**Comment by ANSHUMAN87** [comment]
> Sorry, one last request, based on timing. We are in the middle of releasing the next version of TFF v0.3.0, can you update these links to reflect that new version? Sorry, I thought we would be able to slide this change into the last release. I'll approve after that release. I don't think there would be any merge conflicts so I don't expect any other work there.



> > Sorry, one last request, based on timing. We are in the middle of releasing the next version of TFF v0.3.0, can you update these links to reflect that new version? Sorry, I thought we would be able to slide this change into the last release. I'll approve after that release. I don't think there would be any merge conflicts so I don't expect any other work there.
> 
> No problem, i can do that, would you be able to share one example link like you shared before, so that i can replace with proper one, otherwise there might be chance of broken link. Thanks!

https://github.com/tensorflow/federated/blob/v0.3.0/docs/tutorials/federated_learning_for_text_generation.ipynb  --> Is this correct?

**Comment by michaelreneer** [comment]
> > Sorry, one last request, based on timing. We are in the middle of releasing the next version of TFF v0.3.0, can you update these links to reflect that new version? Sorry, I thought we would be able to slide this change into the last release. I'll approve after that release. I don't think there would be any merge conflicts so I don't expect any other work there.
> 
> > > Sorry, one last request, based on timing. We are in the middle of releasing the next version of TFF v0.3.0, can you update these links to reflect that new version? Sorry, I thought we would be able to slide this change into the last release. I'll approve after that release. I don't think there would be any merge conflicts so I don't expect any other work there.
> > 
> > 
> > No problem, i can do that, would you be able to share one example link like you shared before, so that i can replace with proper one, otherwise there might be chance of broken link. Thanks!
> 
> 
https://github.com/tensorflow/federated/blob/v0.3.0/docs/tutorials/federated_learning_for_text_generation.ipynb --> Is this correct?

Seems right. I don't think the link is broken BTW, we have already pushed the tag for the release, https://github.com/tensorflow/federated/releases/tag/v0.3.0. So the links should all still work. Looks like the link you pasted for federated_learning_for_text_generation.ipynb is perfect.

**Comment by ANSHUMAN87** [comment]
@michaelreneer : I have updated from v0.2.0 --> v0.3.0, please check, Thanks!

**Comment by ANSHUMAN87** [comment]
@michaelreneer : Gentle reminder!

**Comment by michaelreneer** [comment]
@ANSHUMAN87 We have not forgotten about this, I am working on resolving some internal changes are make merging this PR difficult.

**Comment by cramertj** [comment]
Sorry for the long delay on this PR. Unfortunately,  I don't think we can merge this for now. These files are used to generated documentation on the tensorflow.org site, and when those links are used there they should link to the corresponding relative location on tensorflow.org, not to the Github preview of these files.

I did notice that the behavior here is unique to links coming from `.ipynb` files. That is, relative links work just fine coming from a `.md` file. If you're interested in seeing this resolved, I'd file an issue against the `.ipynb` preview support that Github offers.

Thanks for the PR!

**Pull request closed without merging**